modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerIndexForceRebuildTest.java

```diff
@@ -339,9 +344,11 @@ public void testAsyncIndexesRebuild() throws IgniteInterruptedCheckedException {
 
     /**
      * Checks how index force rebuild command behaves when caches are under load.
+     *
+     * @throws Exception If failed.
      */
     @Test
-    public void testIndexRebuildUnderLoad() throws IgniteInterruptedCheckedException {
+    public void testIndexRebuildUnderLoad() throws Exception {
```

Comment:
1. I can`t find any indexes in test caches, did i miss something ? 
2. Do we have any guarantee that "indexes_force_rebuild" still would be in progress while we calling :
cache1.put and ignite.destroyCache(cacheName2) ? 
3. seems we can shrink test a bit if  cahcnge :
destroyCacheLatch.countDown(); and ignite.destroyCache(cacheName2); remove sleep and simple wait destroyCacheLatch ?